src/main/java/com/google/sps/servlets/ActionsServlet.java

```diff
@@ -20,11 +20,12 @@
 @WebServlet("/actions")
 public class ActionsServlet extends HttpServlet {
 
-  private static final String API_KEY = "API_KEY";  // Insert the API_KEY here for testing.
+  private static final String API_KEY = "a752ec3e-9fcf-4500-9107-694351adc5ee";  // Insert the API_KEY here for testing.
   private static final String API_PATH = "https://api.globalgiving.org/api/public/services/search/projects";
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // TODO: Use key to grab keywords from datastore, or use Keywords class.
```

Comment:
The key value in here refers to the keywords themselves (refer to the javascript to see what is being passed to this servlets). Actions and News (Books) servlets shouldn't be accessing datstore, all the keywords are already being passed as query parameters. This way we can save the keywords in the client side, and just get the update information without having the latency of accessing datastore.

Reply:
Sending the keywords to the servlets is already being handled by the fontend.